package.json

```diff
@@ -1,137 +1,129 @@
-        "style-loader": "^0.18.2",
-        "terser-webpack-plugin": "^4.2.2",
-        "webpack": "^4.29.6",
-        "webpack-bundle-analyzer": "^3.3.2",
-        "webpack-cli": "^3.2.3",
-        "webpack-dev-server": "^3.2.1",
-        "webpack-merge": "^4.1.0"
-    },
-    "dependencies": {
-        "@babel/node": "^7.10.5",
-        "@babel/polyfill": "^7.10.4",
-        "@fortawesome/fontawesome-svg-core": "^1.2.15",
-        "@fortawesome/free-brands-svg-icons": "^5.13.0",
-        "@fortawesome/free-regular-svg-icons": "^5.13.0",
-        "@fortawesome/free-solid-svg-icons": "^5.13.0",
-        "@fortawesome/react-fontawesome": "^0.1.4",
-        "accounting": "^0.4.1",
-        "autoprefixer": "^6.7.7",
-        "axios": "^0.21.1",
-        "babel-jest": "^24.6.0",
-        "clean-webpack-plugin": "^0.1.16",
-        "clipboard": "^1.6.1",
-        "commonmark": "^0.27.0",
-        "core-js": "^3.0.1",
-        "d3-hierarchy": "^1.1.4",
-        "d3-interpolate": "^1.1.4",
-        "d3-sankey": "0.7.1",
-        "d3-sankey0.12.3": "npm:d3-sankey@0.12.3",
-        "d3-scale": "^1.0.4",
-        "data-transparency-ui": "github:fedspendingtransparency/data-transparency-ui#v2.3.0",
-        "date-fns": "^2.16.1",
-        "file-loader": "^3.0.1",
-        "fixed-data-table": "0.6.3",
-        "history": "^4.6.1",
-        "immutable": "^3.8.1",
-        "js-cookie": "^2.1.0",
-        "js-search": "^1.4.2",
-        "lodash": "^4.17.12",
-        "mapbox-gl": "^1.13.0",
-        "moment": "2.19.3",
-        "moment-timezone": "^0.5.4",
-        "mousetrap": "^1.6.1",
-        "node-uuid": "^1.4.7",
-        "path-to-regexp": "^1.7.0",
-        "prop-types": "^15.7.2",
-        "r-dom": "^2.3.2",
-        "react": "^16.8.6",
-        "react-addons-perf": "15.4.2",
-        "react-aria-modal": "^2.7.0",
-        "react-checkbox-tree": "1.5.1",
-        "react-copy-to-clipboard": "^5.0.1",
-        "react-custom-scrollbars": "^4.1.2",
-        "react-day-picker": "^7.0.2",
-        "react-dnd-html5-backend": "^2.4.1",
-        "react-dom": "^16.8.6",
-        "react-helmet": "^5.0.0",
-        "react-hot-loader": "^4.8.2",
-        "react-map-gl": "^6.1.13",
-        "react-markdown": "^2.5.0",
-        "react-redux": "^7.0.1",
-        "react-router-dom": "^5.1.2",
-        "react-string-replace": "^0.4.0",
-        "react-transition-group": "^4.4.1",
-        "redux": "^4.0.1",
-        "redux-perf-middleware": "^1.2.2",
-        "regenerator-runtime": "^0.13.2",
-        "svg4everybody": "^2.0.3",
-        "tinycolor2": "^1.3.0",
-        "tunnel": "^0.0.6",
-        "xmldoc": "^0.4.0"
-    }
+  "name": "usaspending-website",
+  "version": "2.5.0",
+  "description": "USAspending Website",
+  "repository": "fedspendingtransparency/usaspending-website",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "npx webpack serve --hot-only  --open --history-api-fallback --progress --config ./webpack/webpack.dev.config.js",
```

Comment:
looks like there are two spaces between --hot-only and --open

Reply:
ah thank you sir